artifacts/rhdh-plugins-reference/argocd/argocd-plugin-admin.adoc

```diff
@@ -66,11 +66,13 @@ global:
     plugins:
       - package: ./dynamic-plugins/dist/roadiehq-backstage-plugin-argo-cd-backend-dynamic
         disabled: false
-      - package: ./dynamic-plugins/dist/janus-idp-backstage-plugin-argocd
+      - package: ./dynamic-plugins/dist/backstage-community-plugin-redhat-argocd
         disabled: false
 ----
 
 [role="_additional-resources"]
 .Additional resources
 
-* For more information on dynamic plugins, see link:https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/configuring_plugins_in_red_hat_developer_hub/rhdh-installing-dynamic-plugins[Dynamic plugin installation].
+* The package path, scope, and name of the {company} ArgoCD plugin has changed since 1.2. For more information, see link:{LinkReleaseNotes}#removed-functionality-rhidp-4293[Breaking Changes] in the _{rn-product-title}_. 
```

Comment:
```suggestion
* The package path, scope, and name of the {company-name} ArgoCD plugin has changed since 1.2. For more information, see link:{LinkReleaseNotes}#removed-functionality-rhidp-4293[Breaking Changes] in the _{rn-product-title}_. 
```